package.json

```diff
@@ -293,6 +293,32 @@
             "default": "",
             "description": "Specifies absolute path of mvn executable.",
             "scope": "window"
+          },
+          "maven.terminal.use.java.home": {
+            "type": "boolean",
+            "default": false,
+            "description": "If true, and if the setting java.home has a value, then the environment variable JAVA_HOME will be set to the value of java.home when a new terminal window is created.",
+            "scope": "window"
+          },
+          "maven.terminal.customEnv": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "title": "environment setting",
+              "properties": {
+                "environmentVariable": {
+                  "type": "string",
+                  "description": "environment variable to set"
+                },
+                "value" : {
+                  "type":"string",
+                  "description": "value for the variable"
+                }
+              }              
+            },
+            "default": [],
+            "description": "An array of environment settings.  These will be added to the terminal session before maven is first run",
```

Comment:
Please improve the wording. E.g. there are 2 spaces before the word "These" now, and missing period in the end.